base/compiler/optimize.jl

```diff
@@ -32,6 +32,20 @@ const IR_FLAG_NOTHROW     = 0x01 << 5
 
 const TOP_TUPLE = GlobalRef(Core, :tuple)
 
+# This corresponds to the type of `CodeInfo`'s `inlining_cost` field
+const INLINE_COST_TYPE = UInt16
```

Comment:
```suggestion
const InlineCostType = UInt16
```
We usually define type-alias with a camel case.